File: utils/timer_utils/golang/timer.go
package timer

import (
	"time"
	"fmt"
)

type Timer struct {
	start time.Time
} 

func NewTimer() *Timer {
    return &Timer{}
}

func (t *Timer) Start() {
	t.start = time.Now()
}

func (t *Timer) Stop(message string) {
	elapsed := time.Since(t.start).Milliseconds()
	fmt.Printf("%s: %d ms \n", message, elapsed)
}

File: utils/timer_utils/python/timer.py
import time

class Timer:
  def __init__(self):
    self.start_time = None

  def start(self):
    self.start_time = time.time()

  def stop(self, message):
    elapsed = (time.time() - self.start_time)
    print(f"{message}: {elapsed: .2f} ms")